src/main/java/utils/NumberUtils.java

```diff
@@ -0,0 +1,21 @@
+package utils;
+
+import java.util.Random;
+
+public class NumberUtils {
+
+	private static final Random random = new java.util.Random();
```

Comment:
```suggestion
	private static final Random random = new Random();
```

`Random` 객체를 `static`으로 선언하셨군요!

이때 `Random` 객체가 반환하는 `nextInt` 메서드는 내부적으로 어떻게 동작하고 있을까요?
> 코드가 어렵다면, 여기서 바로 GPT의 힘을 빌려보도록 해요!

그리고 `Random` 클래스의 Javadoc 마지막 문단에는 어떠한 설명이 되어 있는지 살펴보면 좋을 것 같아요!